### ISSUE TITLE
Make each embed has fixed height

### ISSUE THREAD

**Issue by Corouna** [sugarshin/react-instagram-embed#212]
As titled,

Is this possible?

Sometimes the media contents of the post causing the embed to become varying in heights.. it look especially weird and ugly if i show it as a carousel of Instagram post.

Is there a way to standardize the embed height? Thanks

**Comment by inbaeyoonkr** [comment]
hello @Corouna. Currently i'm woking on with react-instagram-embed and i faced the same problem as you. Did you find the way to get fiexed height?

**Comment by inbaeyoonkr** [comment]
i think the different height because of the caption. hideCaption={true} will give you the same height.

**Comment by ankusharoraa** [comment]
no, height is still varying with hideCaption={true} on multiple posts
